src/Tests/EmbedButtonAdminTest.php

```diff
@@ -99,4 +100,40 @@ public function testEmbedButtonAdmin() {
     $this->assertNoText($name, 'Test embed_button does not appears on the list page');
   }
 
+  /**
+   * Tests the embed_button and file usage integration.
+   */
+  public function testEmbedButtonIcon() {
```

Comment:
Right, but EmbedButtonCrudTest is a KernelTest and the test needs the db schema from file module, entity schema for file and user (because file has a dependency on user). Not sure if it worth the trouble. But sure i can split up the test and add a new testclass.